Resources/Prototypes/_RMC14/Entities/Structures/Machines/Vending/Crew/senior_enlisted_advisor.yml

```
@@ -0,0 +1,106 @@
+# Senior Enlisted Advisor
+#TODO Add Choices to SEA Vendors
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechEquipmentSEA
+  name: ColMarTech SEA equipment rack
+  description: An automated rack hooked up to a colossal storage of Senior Enlisted Advisor standard-issue equipment.
+  suffix: SEA
+  components:
+  - type: AccessReader
+    access: [["CMAccessSEA"]]
+  - type: Sprite
+    sprite: _RMC14/Structures/Machines/VendingMachines/ColMarTech/equipment.rsi
+  - type: CMAutomatedVendor
+    sections:
+    - name: Standard Equipment
+      takeAll: CMStandard
+      entries:
+      - id: CMJumpsuitBO
+      - id: RMCHeadsetSEA
+      - id: CMSatchel
+      - id: CMBootsBlackFilled
+      - id: CMMRE
+    - name: Gloves
+      choices: { CMGloves: 1 }
+      entries:
+      - id: CMHandsBlackMarine
+      - id: CMHandsInsulated
+    - name: Belt
+      choices: { CMBelt: 1 }
+      entries:
+      - id: RMCBeltUtilityGeneral
+      - id: CMBeltUtilityFilled
+      - id: CMBeltMedicBagFilled
+      - id: CMBeltUtilityCombatFilled
+    - name: Pouches
+      choices: { CMPouches: 2 }
+      entries:
+      - id: RMCPouchAutoinjectorFill
+      - id: RMCPouchGeneralLarge
+      - id: RMCPouchMagazineLarge
+      - id: RMCPouchMagazinePistolLarge
+      - id: RMCPouchShotgun #Change to Large when released
+      #Medical Kit Pouch
+      - id: RMCPouchPistol
+      - id: RMCPouchSling
+    - name: Armor
+      choices: { CMArmor: 1 }
+      entries:
+      - id: CMArmorM3VLBallistics
+      - id: CMArmorM3Light
+      - id: CMArmorM3Medium
+      #Bulletproof Vest
+      #USCM Service Jacket
+    - name: Eyewear
+      choices: { CMEyewear: 1 }
+      entries:
+      #Welding Goggles
+      - id: RMCGlassesMedicalHUDGlasses
+      #Prescription Medical HUD Glasses
+      - id: CMGlassesSecurity
+      - id: RMCGlassesMarineRpg #Sunglasses Placeholder
+      #Sunglasses
+    - name: Accessories
+      choices: { CMAccessories: 1 }
+      entries:
+      - id: CMWebbingBrown
+      - id: CMWebbingBlack
+      #Shoulder Holster
+      - id: CMWebbing
+      - id: CMWebbingPouch
+    - name: Headwear
+      choices: { CMHeadwear: 1 }
+      entries:
+      - id: CMHeadCapDrill
+      - id: ArmorHelmetM10
+
+
+- type: entity
+  parent: ColMarTechBase
+  id: ColMarTechGunsSEA
+  name: ColMarTech SEA weapon rack
+  description: An automated rack hooked up to a colossal storage of Senior Enlisted Advisor standard-issue equipment.
+  suffix: SEA
+  components:
+  - type: AccessReader
+    access: [["CMAccessSEA"]]
+  - type: Sprite
+    sprite: _RMC14/Structures/Machines/VendingMachines/ColMarTech/guns.rsi
+  - type: CMAutomatedVendor
+    sections:
+    - name: Personal Primary
+      choices: { CMPrimary: 1 }
+      entries:
+      - id: WeaponRifleM54C
+    - name: Ammunition
+      entries:
+      - id: CMMagazineRifleM54C
+        amount: 10
```

This costs points, it's not a fixed amount
```suggestion
        points: 9
```